config/initializers/doorkeeper_scopes.rb

```diff
@@ -0,0 +1,16 @@
+Doorkeeper::OAuth::PreAuthorization.class_eval do
+  alias_method :old_validate_scopes, :validate_scopes
+
+  def scope
+    @scope.presence || build_scopes
+  end
+
+  def validate_scopes
+    return true if scope.blank?
+    old_validate_scopes
+  end
+
+  def validate_params
+    response_type.blank? ? :response_type : true
+  end
+end
```

Comment:
How does this change here relate to https://github.com/alphagov/signon/pull/1358/commits/30c5f2b1dbc19e34e620399edfa8215bc4970da6 ? Seems to me that they might be related, but I'm unsure?

Reply:
It's replacing this method which rejects a request if scope params are blank
https://github.com/doorkeeper-gem/doorkeeper/blob/5795bc26627d3899ae813ecb5fa64289f5345e45/lib/doorkeeper/oauth/pre_authorization.rb#L95

As https://github.com/alphagov/signon/pull/1358/commits/30c5f2b1dbc19e34e620399edfa8215bc4970da6 sets all `scope`s  as empty strings, we essentially want to ensure that a blank `scope` param is still valid.